plugins/com.python.pydev.analysis/src/com/python/pydev/analysis/refactoring/quick_fixes/AddTokenAndImportStatement.java

```diff
@@ -247,6 +253,47 @@ public void createTextEdit(ComputedInfo computedInfo) {
         }
     }
 
+    // function to get the amount of commas at the end of import string
+    private int verifyComma(String line) {
```

Comment:
I think that you don't want just the comma amount, but the number of whitespaces/tabs too.